agent/src/index.ts

```diff
@@ -418,6 +419,12 @@ export async function createAgent(
                 getSecret(character, "WALLET_PUBLIC_KEY")?.startsWith("0x"))
                 ? evmPlugin
                 : null,
+            (getSecret(character, "SOLANA_PUBLIC_KEY") ||
```

Comment:
Agent can startup, but will fail if these variables are not defined
- `SOLANA_PRIVATE_KEY`
- `SOLANA_ADMIN_PRIVATE_KEY`

Reply:
I missed _PRIVATE_KEY while referring to the solanaPlugin configuration above. I'll fix it right away.